facts_outputs.Rmd

```diff
@@ -0,0 +1,59 @@
+---
+title: "Comparing FACTS Outputs"
+author: "Ciara Donegan"
+date: "2024-09-19"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_knit$set(root.dir = file.path("data","outputs","fair_results"))
+library(tidyverse)
+library(ggplot2)
+library(ncdf4)
+```
+
+## FACTS Outputs
+
+This document compares the outputs of various FACTS experiments, with the original climate input from the FaIR simple climate model and with added input from Hector *(to be added)*.
+
+```{r coupling.ssp585, echo=FALSE}
+# coupling.ssp585 output with FaIR data -- total sea level rise, workflow wf1f
+nc_fslr <- nc_open(file.path("coupling.ssp585","coupling.ssp585.total.workflow.wf1f.global.nc"))
+f_slr <- ncvar_get(nc_fslr,"sea_level_change")
+f_years <- ncvar_get(nc_fslr,"years")
+f_samples <- ncvar_get(nc_fslr,"samples")
+nc_close(nc_fslr)
+
+# Convert from wide dataframe format to long
+colnames(f_slr) <- f_samples
+f_slr_df <- as.data.frame(f_slr)
+f_slr_df$year <- f_years
+
+f_slr_long <- f_slr_df %>%
```

Comment:
Nice reshaping data from wide to long another way to get the max, min, and mean values without having to do the reshaping is using the apply function. This has the potential to speed up your code. 

This isn't a necessary change more of an FYI the following chunk of code will return vectors that could be used to make your `f_slr_stats` data.frame  

```
  min <- apply(X = f_slr, MARGIN = 1, FUN = min) 
  max <- apply(X = f_slr, MARGIN = 1, FUN = max) 
  mean <- apply(X = f_slr, MARGIN = 1, FUN = mean) 
```